ivadomed/transforms.py

```diff
@@ -258,26 +259,72 @@ def __call__(self, sample, metadata=None):
         is_2d = sample.shape[-1] == 1
         metadata['preresample_shape'] = sample.shape
         zooms = list(metadata["zooms"])
-
```

Comment:
Please revert this change. There should be no change here.

Reply:
fixed.